README.md

```diff
@@ -105,7 +105,7 @@ We use [Next.js](https://nextjs.org/) as our react framework. This handles build
 
 There are three protected branches in this repository: `dev`, `qa`, and `production`.
 
-- `production` is the most current code, used in production at NYPL. Any time a PR is merged in, a release is tagged and created. A Docker Image is then built by Github Actions, which is finally pushed to Docker Hub with the tag `latest` as well as the specific version tag of the release (ie `2.3.6`). Generally, we only make PRs to `production` from `qa` unless a hotfix is necessary.
+- `production` is the most current code, used in production at LYRASIS. Any time a PR is merged in, a release is tagged and created. A Docker Image is then built by Github Actions, which is finally pushed to Docker Hub with the tag `latest` as well as the specific version tag of the release (ie `2.3.6`). Generally, we only make PRs to `production` from `qa` unless a hotfix is necessary.
```

Comment:
We probably want to say something like "...used in production for The Palace Project.", rather than referencing LYR.

Reply:
Fixed.